### PR TITLE
Added navbar mobile view transition

### LINKED ISSUE #483: Navigation Bar (mobile view) transition
[Reference](https://codepen.io/Zaku/pen/vcaFr)

Remember: we need just the transition keeping the color palette the same

### PR DESCRIPTION
**Pull request by Harshita-Kanal**
### Description
Added transition for mobile view navigation bar. 
Fixes #483 
### Screenshot
John Doe _ Home

### CODE REVIEW

File: assets/css/home.css [hunk 1]
```diff
@@ -266,6 +266,66 @@ a:hover {
   color: transparent;
 }
```
Comment (smaranjitghose) [inline assets/css/home.css:267]: This has to be in ``style.css`` to be called on all other pages as well

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
navvv
Can you please remove the outline or border

**Comment by smaranjitghose** [comment]
@anushbhatia any color related suggestions or perhaps for the on hover transitions?

File: assets/css/home.css [hunk 2]
```diff
@@ -266,6 +266,66 @@ a:hover {
...
+  transition: all .2s ease-in-out; 
+}
+
+.menu-btn__burger{
+  position: absolute;
+  width: 40px;
+  height: 4px;
+  background: #040406;
+  border-radius: 2px;
+  box-shadow: 0 2px 5px rgba(255, 101, 47, .2);
```
Comment (anushbhatia) [inline assets/css/home.css:287]: Add a box shadow of black red looks pretty weird on this page

**Comment by Harshita-Kanal** [comment]
I have made the required changes, I was also able to implement the animation as per the reference given in the issue, please check it out as well, fixed the toggling of the navbar as well in  some pages by adding the jquery and popper cdn links in script section.
John Doe _ Home (1)

File: assets/js/app.js [hunk 3]
```diff
@@ -56,7 +60,21 @@ let footer = $(`
 $(function () {
 	let bodyElement = $(`body`);
 	bodyElement.prepend(header);
-	bodyElement.append(footer);
```
Comment (smaranjitghose) [inline assets/js/app.js:59]: Keep the indentation  constant for all function definations

**Comment by Harshita-Kanal** [comment]
Fixed the indentation.

**Review by anushbhatia — APPROVED** [review]

**Comment by smaranjitghose** [comment]
In a follow up PR, make sure the side bar extends to the entire screen on hover/click as done [here](https://girlscriptboilerplate.netlify.app/)

**Comment by smaranjitghose** [comment]
Great work with clean code @Harshita-Kanal !

**Comment by Harshita-Kanal** [comment]
Thanks 🙂

**Pull request merged**